*** ./smali/com/android/internal/statusbar/StatusBarIcon.smali	2013-08-21 11:30:23.368621851 -0700
--- /home/adit/patchrom/m7ul/temp/new_smali/framework.jar.out/./smali/com/android/internal/statusbar/StatusBarIcon.smali	2013-08-21 11:29:19.693599954 -0700
***************
*** 63,68 ****
--- 63,94 ----
      return-void
  .end method
  
+ .method public constructor <init>(Lcom/android/internal/statusbar/StatusBarNotification;IIILjava/lang/CharSequence;)V
+     .locals 6
+     .parameter "notification"
+     .parameter "iconId"
+     .parameter "iconLevel"
+     .parameter "number"
+     .parameter "contentDescription"
+ 
+     .prologue
+     iget-object v1, p1, Lcom/android/internal/statusbar/StatusBarNotification;->pkg:Ljava/lang/String;
+ 
+     move-object v0, p0
+ 
+     move v2, p2
+ 
+     move v3, p3
+ 
+     move v4, p4
+ 
+     move-object v5, p5
+ 
+     invoke-direct/range {v0 .. v5}, Lcom/android/internal/statusbar/StatusBarIcon;-><init>(Ljava/lang/String;IIILjava/lang/CharSequence;)V
+ 
+     return-void
+ .end method
+ 
  .method public constructor <init>(Ljava/lang/String;IIILjava/lang/CharSequence;)V
      .locals 1
      .parameter "iconPackage"
