*** ./smali/android/view/Window.smali	2013-08-21 11:29:47.086945745 -0700
--- /home/adit/patchrom/m7ul/temp/new_smali/framework.jar.out/./smali/android/view/Window.smali	2013-08-21 11:28:46.149257453 -0700
***************
*** 548,553 ****
--- 548,568 ----
  .method public abstract isFloating()Z
  .end method
  
+ .method public isHardwareAccelerated()Z
+     .locals 1
+ 
+     .prologue
+     invoke-virtual {p0}, Landroid/view/Window;->getWindowManager()Landroid/view/WindowManager;
+ 
+     move-result-object v0
+ 
+     invoke-interface {v0}, Landroid/view/WindowManager;->isHardwareAccelerated()Z
+ 
+     move-result v0
+ 
+     return v0
+ .end method
+ 
  .method public abstract isShortcutKey(ILandroid/view/KeyEvent;)Z
  .end method
  
