*** ./smali/com/android/server/am/AppNotRespondingDialog.smali	2013-08-21 11:30:27.058419487 -0700
--- /home/adit/patchrom/m7ul/temp/new_smali/services.jar.out/./smali/com/android/server/am/AppNotRespondingDialog.smali	2013-08-21 11:29:24.519951309 -0700
***************
*** 300,305 ****
--- 300,317 ----
  
  
  # virtual methods
+ .method getProc()Lcom/android/server/am/ProcessRecord;
+     .locals 1
+     .annotation build Landroid/annotation/MiuiHook;
+         value = .enum Landroid/annotation/MiuiHook$MiuiHookType;->NEW_METHOD:Landroid/annotation/MiuiHook$MiuiHookType;
+     .end annotation
+ 
+     .prologue
+     iget-object v0, p0, Lcom/android/server/am/AppNotRespondingDialog;->mProc:Lcom/android/server/am/ProcessRecord;
+ 
+     return-object v0
+ .end method
+ 
  .method public onStop()V
      .locals 0
  
