*** ./smali/android/app/ActivityManager.smali	2013-08-21 11:30:12.787466807 -0700
--- /home/adit/patchrom/m7ul/temp/new_smali/framework.jar.out/./smali/android/app/ActivityManager.smali	2013-08-21 11:29:09.256515100 -0700
***************
*** 256,261 ****
--- 256,264 ----
  .method public static isHighEndGfx(Landroid/view/Display;)Z
      .locals 8
      .parameter "display"
+     .annotation build Landroid/annotation/MiuiHook;
+         value = .enum Landroid/annotation/MiuiHook$MiuiHookType;->CHANGE_CODE:Landroid/annotation/MiuiHook$MiuiHookType;
+     .end annotation
  
      .prologue
      const/4 v3, 0x1
***************
*** 300,306 ****
  
      if-ge v1, v4, :cond_0
  
!     const/4 v3, 0x0
  
      goto :goto_0
  .end method
--- 303,311 ----
  
      if-ge v1, v4, :cond_0
  
!     invoke-static {p0}, Landroid/app/ExtraActivityManager;->isHighEndGfx(Landroid/view/Display;)Z
! 
!     move-result v3
  
      goto :goto_0
  .end method
