*** ./smali/com/android/server/LightsService.smali	2013-08-21 11:30:27.862811379 -0700
--- /home/adit/patchrom/m7ul/temp/new_smali/services.jar.out/./smali/com/android/server/LightsService.smali	2013-08-21 11:29:25.280321754 -0700
***************
*** 207,209 ****
--- 207,225 ----
  
      return-object v0
  .end method
+ 
+ .method setLight(ILcom/android/server/LightsService$Light;)V
+     .locals 1
+     .parameter "id"
+     .parameter "light"
+     .annotation build Landroid/annotation/MiuiHook;
+         value = .enum Landroid/annotation/MiuiHook$MiuiHookType;->NEW_METHOD:Landroid/annotation/MiuiHook$MiuiHookType;
+     .end annotation
+ 
+     .prologue
+     iget-object v0, p0, Lcom/android/server/LightsService;->mLights:[Lcom/android/server/LightsService$Light;
+ 
+     aput-object p2, v0, p1
+ 
+     return-void
+ .end method
