*** ./smali/com/android/internal/telephony/IIccPhoneBook.smali	2013-08-21 11:30:22.780335243 -0700
--- /home/adit/patchrom/m7ul/temp/new_smali/framework.jar.out/./smali/com/android/internal/telephony/IIccPhoneBook.smali	2013-08-21 11:29:19.025274352 -0700
***************
*** 15,20 ****
--- 15,28 ----
  
  
  # virtual methods
+ .method public abstract getAdnCapacity()I
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
  .method public abstract getAdnRecordsInEf(I)Ljava/util/List;
      .annotation system Ldalvik/annotation/Signature;
          value = {
***************
*** 37,42 ****
--- 45,58 ----
      .annotation system Ldalvik/annotation/Throws;
          value = {
              Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
+ .method public abstract getFreeAdn()I
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
          }
      .end annotation
  .end method
